<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Influenza diversity across spatial scales</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	
	<!-- Customized Bootstrap + Font Awesome + Solarized -->
	<link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/Logos/lab_header_logo.png" sizes="16x16" type="image/png">

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	<!-- Google Analytics -->
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-26244371-2', 'bedford.io');
		ga('send', 'pageview');

	</script>

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

	<!-- FONT -->
	<link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block ml-0 mr-1" src="/images/Logos/lab_header_logo.png">
					IBH Lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/About/">About</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">Team</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/research/">Research</a></li>
						<!-- 
						<li class="nav-link">
						
						<a class="mx-1" href="/Publication/">Publication</a></li> -->
						
						<li class="nav-link">
						
						<a class="mx-1" href="/join us/">Join Us!</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container" style="margin-top: 7rem;">

	<div class="blog-post">
	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			<div class="media">
				<div class="media-body">
					<div class="title media-heading">
						<a href="/about/influenza-across-spatial-scales/" class="off">Influenza diversity across spatial scales</a>
					</div>
					<p>
					<div class="smallhead">
						26 Feb 2013
						
						by <a class="off" href="
							
						">
						Trevor Bedford
						</a>
						
						
					</div>
				</div>
			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>

	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			
			<div class="post">
				<p>It’s clear that there exists <a href="/blog/global-flu-diversity/">substantial global genetic diversity in influenza at any given moment</a>.  This level of diversity could be reflected in diverse local epidemics, or local epidemics could be homogeneous with specific locations encountering only a single strain each season, even while maintaining substantial diversity at the global level. This is an important distinction if we care about how local populations react to influenza epidemics and in assessing vaccine effectiveness. Here, I’ve sought to quantify this distinction.</p>

<p>I began by collecting all the sequences I could for influenza A/H3N2 from GenBank from 1995 to 2011, keeping only sequences with complete dates. Because we’re interested in functional diversity, I reduced this to only amino acid sequences from the HA1 protein and further reduced these sequences to 129 epitope sites following <a href="http://www.sciencedirect.com/science/article/pii/S0264410X0400636X">Munoz and Deem (2005)</a>. These epitope sites represent residues that when mutated often result in proteins that elicit a novel immune response, and hence mutations to these sites often provide a transmission advantage to the virus. I removed sequences which possessed gaps at any of these 129 sites, leaving a total of 8482 sequences to work from.</p>

<p><img src="/images/blog/theta_by_scale.png" alt="" /></p>

<p>I next calculated the number of unique variants present for each Northern Hemisphere influenza season (October to March), giving the relationship between sample count and number of unique sequence types shown here. I calculated this relationship for the entire global sample, for the USA, Europe, Japan and Korea (Northern), for just the USA and within each state in the US.  This relationship can be quantified with the <a href="http://en.wikipedia.org/wiki/Ewens's_sampling_formula">Ewen’s sampling formula</a>, giving an expectation for the number of unique variants <i>k</i> observed in a sample of <i>n</i> sequences</p>
<div style="text-align: center; margin-top: 0.5em; margin-bottom: 0.5em;"><span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mstyle scriptlevel="0" displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mfrac><mi>θ</mi><mrow><mi>θ</mi><mo>+</mo><mi>i</mi><mo>−</mo><mn>1</mn></mrow></mfrac><mo separator="true">,</mo></mstyle></mrow><annotation encoding="application/x-tex">\displaystyle \sum_{i=1}^n \frac{\theta}{\theta +i - 1},</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:2.929066em;vertical-align:-1.277669em;"></span><span class="mop op-limits"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.6513970000000002em;"><span style="top:-1.872331em;margin-left:0em;"><span class="pstrut" style="height:3.05em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathnormal mtight">i</span><span class="mrel mtight">=</span><span class="mord mtight">1</span></span></span></span><span style="top:-3.050005em;"><span class="pstrut" style="height:3.05em;"></span><span><span class="mop op-symbol large-op">∑</span></span></span><span style="top:-4.3000050000000005em;margin-left:0em;"><span class="pstrut" style="height:3.05em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathnormal mtight">n</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.277669em;"><span></span></span></span></span></span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.37144em;"><span style="top:-2.314em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathnormal" style="margin-right:0.02778em;">θ</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mord mathnormal">i</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mord">1</span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathnormal" style="margin-right:0.02778em;">θ</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.7693300000000001em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mpunct">,</span></span></span></span></span></div>
<p>where <i>θ</i> represents the level of mutational input into the population. I estimate <i>θ</i> from the these relationships getting a global estimate of <i>θ</i> of 58.5 and a within-state estimate of <i>θ</i> of 7.3.  Using the Ewen’s sampling formula, we predict that, in the average influenza season, 100 random infections sampled from the global population will yield 59 unique variants and 100 random infections sampled at the state-level will yield 23 unique variants.</p>

<p>It is common is measure diversity in terms of pairwise identity.  In population genetics this is often referred to as heterozygosity, named because it is the probability that two randomly chosen haploid individuals (or chromosomes within a diploid individual) are distinct.  This is the same measure as 1 minus the <a href="http://en.wikipedia.org/wiki/Diversity_index#Simpson_index">Simpson index</a> (1 - <i>λ</i>) in ecology.  Here, I find that 89% of pairs sampled from the global population are distinct and 72% of pairs sampled from within-state populations are distinct.  Along these lines, it’s common to measure sequence “diversity” or <i>π</i> as the average number of mutations that separate random pairs of sequences.  In this case, I find that pairs of sequences sampled from the global population are separated by 4.1 epitope mutations, and pairs of sequences sampled from within-state populations are separated by 2.6 epitope mutations.  You may have noticed that <i>θ</i> estimated by Ewen’s doesn’t match up with <i>θ</i> estimated from pairwise diversity (58.5 vs. 4.1 and 7.3 vs. 2.6 for global and local samples, respectively).  This is essentially a Ewens-Watterson test showing an overabundance of rare variants, consistent with the actions of natural selection or otherwise non-neutral demography.</p>

<p>So, although there is significantly more genetic variation at the global level, local variation is still substantial.  We expect two people who get the flu within a state (and most likely a community) in the same season to have been infected by distinct viruses.  It’s not clear whether viruses differing at one of these 129 epitope sites are always immunologically distinct, but many of these mutations to epitope sites should have an immunological impact.</p>

			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>
	<div class="spacer"></div>

</div>


	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
