<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Openings for bioinformatics analyst and software engineer to contribute to Nextstrain platform</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	
	<!-- Customized Bootstrap + Font Awesome + Solarized -->
	<link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/Logos/lab_header_logo.png"/>

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	<!-- Google Analytics -->
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-26244371-2', 'bedford.io');
		ga('send', 'pageview');

	</script>

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

	<!-- FONT -->
	<link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block ml-0 mr-1" src="/images/Logos/lab_header_logo.png">
					IBH Lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/About/">About</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/Team/">Team</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/Research/">Research</a></li>
						<!-- 
						<li class="nav-link">
						
						<a class="mx-1" href="/Publication/">Publication</a></li> -->
						
						<li class="nav-link">
						
						<a class="mx-1" href="/join us/">Join Us!</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container" style="margin-top: 7rem;">

	<div class="blog-post">
	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			<div class="media">
				<div class="media-body">
					<div class="title media-heading">
						<a href="/about/2023-nextstrain-hiring/" class="off">Openings for bioinformatics analyst and software engineer to contribute to Nextstrain platform</a>
					</div>
					<p>
					<div class="smallhead">
						21 Jul 2023
						
						by <a class="off" href="
							
						">
						Trevor Bedford
						</a>
						
						
					</div>
				</div>
			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>

	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			
			<div class="post">
				<p><em>These positions have been filled.</em></p>

<p>Positions for a <a href="#bioinformatics-analyst-iiiii">bioinformatics analyst</a> and a <a href="#software-engineer-ii">software engineer</a> are available immediately in the Bedford lab at the Fred Hutch. Details for both positions follow:</p>

<hr />

<h3 id="bioinformatics-analyst-iiiii">Bioinformatics Analyst II/III</h3>

<p>We have an opening for a bioinformatician in the <a href="https://bedford.io/">Bedford lab</a> at the Fred Hutch to work on genomic epidemiology and evolutionary analysis of pathogens such as SARS-CoV-2, seasonal influenza, and <a href="https://nextstrain.org/pathogens">other emerging and endemic pathogens</a>. This position will contribute to ongoing work for the Bedford lab and Nextstrain.</p>

<p><a href="https://nextstrain.org/">Nextstrain</a> is an award-winning project for tracking infectious disease epidemics developed in collaboration with the <a href="https://neherlab.org/">Neher lab</a> at the University of Basel. Nextstrain won the <a href="https://www.nih.gov/news-events/news-releases/open-science-prize-announces-epidemic-tracking-tool-grand-prize-winner">Open Science Prize</a> in Feb 2017 and has been instrumental in analysis of the <a href="https://nextstrain.org/sars-cov-2">SARS-CoV-2 pandemic</a>, <a href="https://nextstrain.org/community/inrb-drc/ebola-nord-kivu">Ebola outbreaks</a>, <a href="https://nextstrain.org/zika/">Zika spread</a> in the Americas and is used by the World Health Organization to aid in the process of <a href="https://nextstrain.org/flu/">influenza vaccine strain selection</a>. The software we write to power <a href="https://docs.nextstrain.org/en/latest/learn/parts.html">all parts</a> of Nextstrain—bioinformatics, visualizations, analysis pipelines, data management, and more—is entirely <a href="https://opensource.org/osd">open-source</a> and <a href="https://github.com/nextstrain">available to the public</a>. We work with public health entities and scientists across the world, both formally and informally, to expand pathogen surveillance capabilities and to improve the automation and robustness of these analyses. Our goal is to empower the wider genomic epidemiology and public health communities to tweak our analyses, create new ones, and communicate scientific insights using the same tools we do.</p>

<h4 id="responsibilities">Responsibilities</h4>

<p>This role advances the research aims of the Bedford lab and the Nextstrain team through a combination of independent work, collaboration with scientists and software developers in the group, and interactions with the wider public health and science communities. In this role, the bioinformatician will:</p>

<ul>
  <li>Develop and maintain analytic pipelines such as those that clean and ingest genome metadata, build phylogenetic trees, and run forecasting models for SARS-CoV-2 and other pathogens</li>
  <li>Improve the robustness, automation, and monitoring of our existing pathogen pipelines</li>
  <li>Develop reproducible pipelines to expand surveillance of endemic and emerging human pathogens, in collaboration with both internal and external groups</li>
  <li>Participate in community outreach through office hours, discussion forums, and mailing lists</li>
  <li>Write and maintain thorough documentation on software and pipelines</li>
  <li>Design software with a diverse range of collaborators and users in mind</li>
  <li>Contribute to the Nextstrain team’s decision-making and planning processes</li>
  <li>Present at Bedford lab meetings</li>
</ul>

<h4 id="qualifications">Qualifications</h4>

<h5 id="minimum-qualifications">Minimum qualifications</h5>

<ul>
  <li>Master’s degree in bioinformatics, computational biology, biology, or related field with at least three years’ direct experience in computational analysis of large sequence-based molecular data sets.</li>
  <li>Fluency in at least one high-level programming language, such as Python, R, Ruby, JavaScript, or Perl</li>
  <li>Familiarity with version control and other software development best practices</li>
  <li>Experience with workflow managers such as Snakemake, Nextflow, or WDL</li>
  <li>Knowledge of molecular biology</li>
  <li>Motivated to learn new skills and technologies and collaborate within an existing team’s practices</li>
  <li>Excellent written and verbal communication skills</li>
</ul>

<h5 id="preferred-qualifications">Preferred qualifications</h5>

<ul>
  <li>Expertise in genomics</li>
  <li>Knowledge of automated testing and workflows such as GitHub Actions</li>
  <li>Experience configuring and deploying analyses on a cloud infrastructure</li>
</ul>

<p>The position is available immediately with flexible starting dates. Informal inquiries are welcome. Applications will be accepted until the position is filled. We offer a competitive salary commensurate with skills and experience, along with benefits. We are committed to improving diversity in the computational sciences. Applicants of diverse backgrounds are particularly encouraged to apply. This is a full-time (40 hours/week) position, but depending on the applicant, could be a salaried employee or contracted hourly consultant. An ideal candidate would be local to the Seattle area or willing to relocate, but remote work is also an option.</p>

<p>To aid in applicant review, a coding sample is requested. We’re happy to review whatever you’re most proud of (in any programming language). If you don’t have code that can be publicly shared, that’s okay. Please apply anyway and just let us know that this isn’t available.</p>

<p>If you think you might be a great fit for this position but are concerned about meeting all qualifications, we’d like to hear from you. Please email Trevor Bedford at tb<span style="display:none">obfuscate</span>@bedford.io and John Huddleston jhuddles<span style="display:none">obfuscate</span>@fredhutch.org.</p>

<p>To apply for this position, please go to the <a href="https://careers-fhcrc.icims.com/jobs/25697/job">official Fred Hutch listing</a>.</p>

<hr />

<h3 id="software-engineer-ii">Software Engineer II</h3>

<p>The <a href="https://bedford.io/">Bedford Lab</a> at the <a href="https://fredhutch.org/">Fred Hutch</a> is seeking a software engineer to work on <a href="https://nextstrain.org/">Nextstrain</a>, an <a href="https://www.nih.gov/news-events/news-releases/open-science-prize-announces-epidemic-tracking-tool-grand-prize-winner">award-winning</a> project for tracking infectious disease epidemics such as the <a href="https://nextstrain.org/sars-cov-2">SARS-CoV-2 pandemic</a>, <a href="https://nextstrain.org/community/inrb-drc/ebola-nord-kivu">Ebola outbreaks</a>, <a href="https://nextstrain.org/zika/">Zika spread</a> in the Americas, <a href="https://nextstrain.org/flu/">seasonal flu</a>, and <a href="https://nextstrain.org/pathogens">other emerging and endemic pathogens</a>. This position will augment our existing team to design, develop, maintain, operate, and support our software and services that empower research scientists and public health practitioners in the lab and around the world.</p>

<p>Nextstrain, developed in collaboration with the <a href="https://neherlab.org/">Neher Lab</a> at the University of Basel, provides tools for evolutionary analysis of pathogens and genomic epidemiology. We write <a href="https://github.com/nextstrain/">open source software</a> in a public development style to power <a href="https://docs.nextstrain.org/en/latest/learn/parts.html">all parts</a> of Nextstrain—bioinformatics, visualizations, analysis pipelines, data management, and more—and our analyses use open data whenever possible. We work with public health entities and scientists across the world, both formally and informally, to expand pathogen surveillance capabilities and to improve the automation and robustness of these analyses. Our goal is to empower the wider genomic epidemiology and public health communities to tweak our analyses, create new ones, and communicate scientific insights using the same tools we do.</p>

<h4 id="about-the-role">About the role</h4>

<p>This position will be responsible for general software engineering and development work across the entire Nextstrain stack. This includes command-line applications for bioinformatics and data/workflow management (e.g. <a href="https://github.com/nextstrain/augur">Augur</a>, <a href="https://github.com/nextstrain/cli">Nextstrain CLI</a>), visualization applications for phylogenetics (e.g. <a href="https://github.com/nextstrain/auspice">Auspice</a>), full-stack web applications for sharing analyses (e.g. <a href="https://github.com/nextstrain/nextstrain.org">nextstrain.org</a>), workflows for data curation and analysis (e.g. <a href="https://github.com/nextstrain/ncov-ingest/">ncov-ingest</a>), runtimes for Nextstrain analyses (e.g. <a href="https://github.com/nextstrain/docker-base">docker-base</a>, <a href="https://github.com/nextstrain/conda-base">conda-base</a>), and internal tooling/infrastructure to support all of that.</p>

<h4 id="what-we-provide">What we provide</h4>

<ul>
  <li>Empowerment to craft software that helps protect the world from epidemics and pandemics</li>
  <li>Thrive in an ecosystem of cross-disciplinary learning, drawing insights from scientists, public health practitioners, and fellow software developers</li>
  <li>A team that believes in continuous learning and cultivates an environment where all members of the group help each other</li>
  <li>Opportunity for growth as a software developer in areas of personal interest (e.g. front-end JavaScript, back-end infrastructure, data pipelines, being a project lead, etc.)</li>
  <li>A team culture that champions a healthy work life balance</li>
  <li>A competitive compensation package, with comprehensive health and welfare benefits</li>
</ul>

<h4 id="what-youll-do">What you’ll do</h4>

<ul>
  <li>Design, develop, test, document, and maintain software under a coherent ecosystem</li>
  <li>Release new versions of packaged programs for installation by users and deploy new versions of hosted services to users</li>
  <li>Configure and manage cloud infrastructure resources (e.g. AWS, Heroku, Terraform)</li>
  <li>Create, extend, and troubleshoot automated workflows (e.g. GitHub Actions, Snakemake, Nextflow, WDL)</li>
  <li>Participate in constructive code review processes with other team members</li>
  <li>Support internal and external users of software projects via various communication channels</li>
</ul>

<p>Integrating with an existing team both in-person and online is a key aspect of this position. This position will work daily within a small team of Bedford Lab members and collaborators. The Nextstrain team communicates openly about project and organizational decisions and encourages participation by all team members in decision-making.</p>

<h4 id="minimum-qualifications-1">Minimum qualifications</h4>

<ul>
  <li>3+ years of experience in software engineering</li>
  <li>Fluency in Python and JavaScript/TypeScript, or fluency in similar languages</li>
  <li>Proficiency with Linux/Unix and command-line interfaces</li>
  <li>Proficiency with version control and software development best practices</li>
  <li>Excellent written and verbal communication skills</li>
  <li>Motivation to learn and collaborate within an existing team’s practices</li>
</ul>

<p>The position is available immediately with flexible starting dates. Informal inquiries are welcome. Applications will be accepted until the position is filled. We offer a competitive salary commensurate with skills and experience, along with benefits. We are committed to improving diversity in the computational sciences. Applicants of diverse backgrounds are particularly encouraged to apply. This is a full-time (40 hours/week) position, but depending on the applicant, could be a salaried employee or contracted hourly consultant. An ideal candidate would be local to the Seattle area or willing to relocate, but remote work is also an option.</p>

<p>To aid in applicant review, we request you submit a cover letter, your resume, and a coding sample. For the coding sample, we’re happy to review whatever you’re most proud of (in any programming language). If you don’t have code that can be publicly shared, that’s okay. Please apply anyway and just let us know that this isn’t available.</p>

<p>If you’re interested in this position but are concerned about meeting all the qualifications, we’d like to hear from you. Please email Trevor Bedford at tb<span style="display:none">obfuscate</span>@bedford.io and Thomas Sibley at tsibley<span style="display:none">obfuscate</span>@fredhutch.org.</p>

<p>To apply for this position, please go to the <a href="https://hhmi.wd1.myworkdayjobs.com/en-US/External/job/Software-Engineer-II--Bedford-Lab_R-2176">official HHMI listing</a>.</p>

			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>
	<div class="spacer"></div>

</div>


	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
