<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Cryptic transmission of novel coronavirus revealed by genomic epidemiology</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	
	<!-- Customized Bootstrap + Font Awesome + Solarized -->
	<link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/Logos/lab_header_logo.png"/>

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	<!-- Google Analytics -->
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-26244371-2', 'bedford.io');
		ga('send', 'pageview');

	</script>

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

	<!-- FONT -->
	<link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block ml-0 mr-1" src="/images/Logos/lab_header_logo.png">
					IBH Lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/About/">About</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">Team</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/research/">Research</a></li>
						<!-- 
						<li class="nav-link">
						
						<a class="mx-1" href="/Publication/">Publication</a></li> -->
						
						<li class="nav-link">
						
						<a class="mx-1" href="/join us/">Join Us!</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container" style="margin-top: 7rem;">

	<div class="blog-post">
	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			<div class="media">
				<div class="media-body">
					<div class="title media-heading">
						<a href="/about/ncov-cryptic-transmission/" class="off">Cryptic transmission of novel coronavirus revealed by genomic epidemiology</a>
					</div>
					<p>
					<div class="smallhead">
						2 Mar 2020
						
						by <a class="off" href="
							
						">
						Trevor Bedford
						</a>
						
						
					</div>
				</div>
			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>

	<div class="row">
		<div class="col-lg-1"></div>
		<div class="col-lg-10">
			
			<div class="post">
				<p>The field of genomic epidemiology focuses on using the genetic sequences of pathogens to understand patterns of transmission and spread.
Viruses mutate very quickly and accumulate changes during the process of transmission from one infected individual to another.
The novel coronavirus which is responsible for the emerging COVID-19 pandemic mutates at an average of about <a href="http://virological.org/t/phylodynamic-analysis-129-genomes-24-feb-2020/356">two mutations per month</a>.
After someone is exposed they will generally incubate the virus for ~5 days before symptoms develop and transmission occurs.
Other research has shown that the “serial interval” of SARS-CoV-2 is ~7 days.
You can think of a transmission chain as looking something like:</p>

<p><br />
<img src="/images/blog/ncov_transmission_chain_simple.png" alt="" />
<br /></p>

<p>where, on average, we have 7 days from one infection to the next.
As the virus transmits, it will mutate at this rate of two mutations per month.
This means, that on average every other step in the transmission chain will have a mutation and so would look something like:</p>

<p><br />
<img src="/images/blog/ncov_transmission_chain_mutation.png" alt="" />
<br /></p>

<p>These mutations are generally really simple things.
An ‘A’ might change to a ‘T’, or a ‘G’ to a ‘C’.
This changes the genetic code of the virus, but it’s hard for a single letter change to do much to make the virus behave differently.
However, with advances in technology, it’s become readily feasible to sequence the genome of the novel coronavirus.
This works by taking a swab from someone’s nose and extracting the RNA in the sample and then determining the ‘letters’ of this RNA genome using <a href="https://en.wikipedia.org/wiki/Illumina_dye_sequencing">chemistry and very powerful cameras</a>.
Each person’s coronavirus infection will yield a sequence of 30,000 ‘A’, ‘T’, ‘G’ or ‘C’ letters.
We can use these sequences to reconstruct which infection is connected to which infection.
As an example, if we sequenced three of these infections and found:</p>

<p><br />
<img src="/images/blog/ncov_transmission_chain_sequenced.png" alt="" />
<br /></p>

<p>We could take the “genomes” ATTT, ATCT and GTCT and infer that the infection with sequence ATTT lead to the infection with sequence ATCT and this infection lead to the infection with sequence GTCT.
This approach allows us learn about epidemiology and transmission in a completely novel way and can supplement more traditional contact tracing and case-based reporting.</p>

<p>For a few years now, we’ve been working on the <a href="https://nextstrain.org/">Nextstrain</a> software platform, which aims to make genomic epidemiology as rapid and as useful as possible.
We had previously applied this to outbreaks like <a href="https://nextstrain.org/ebola">Ebola</a>, <a href="https://nextstrain.org/zika">Zika</a> and <a href="https://nextstrain.org/flu/seasonal/h3n2/ha/2y">seasonal flu</a>.
Owing to advances in technology and open data sharing, the genomes of 140 SARS-CoV-2 coronaviruses have been shared from all over the world via <a href="https://gisaid.org">gisaid.org</a>.
As these genomes are shared, we download them from GISAID and <a href="/blog/genomic-epi-for-ncov-response/">incorporate them into a global map as quickly as possible</a> and have an always up-to-date view of the genomic epidemiology of novel coronavirus at <a href="https://nextstrain.org/ncov">nextstrain.org/ncov</a>.</p>

<p>The big picture looks like this at the moment:</p>

<p><br />
<img src="/images/blog/ncov_nextstrain_2020_03_01.png" alt="" />
<br /></p>

<p>where we can see the earliest infections in Wuhan, China in purple on the left side of the tree.
All these genomes from Wuhan have a common ancestor in late Nov or early Dec, suggesting that this virus has emerged recently in the human population.</p>

<p>The first case in the USA was called “USA/WA1/2020”.
This was from a <a href="https://www.seattletimes.com/seattle-news/health/case-of-wuhan-coronavirus-detected-in-washington-state-first-in-united-states/">traveller directly returning from Wuhan to Snohomish County on Jan 15</a>, with a swab collected on Jan 19.
This virus was rapidly sequenced by the US CDC Division of Viral Diseases and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN985325">shared publicly</a> on Jan 24 (huge props to the CDC for this).
We can zoom into the tree to place WA1 among related viruses:</p>

<p><br />
<img src="/images/blog/ncov_nextstrain_2020_03_01_wa1.png" alt="" />
<br /></p>

<p>The virus has an identical genome to the virus Fujian/8/2020 sampled in Fujian on Jan 21, also labeled as a travel export from Wuhan, suggesting a close relationship between these two cases.</p>

<p>Last week the <a href="https://seattleflu.org/">Seattle Flu Study</a> started screening samples for COVID-19 as described <a href="https://twitter.com/trvrb/status/1233868710156234752">here</a>.
Soon after starting screening <a href="https://www.snohd.org/Blog.aspx?IID=13">we found a first positive in a sample from Snohomish County</a>.
The case was remarkable in that it was a “community case”, only the second recognized in the US, someone who had sought treatment for flu-like symptoms, been tested for flu and then sent home owing to mild disease.
After this was diagnostically confirmed by Shoreline Public Health labs on Fri Feb 28 we were able to immediately get the sample USA/WA2/2020 on a sequencer and have a genome available on Sat Feb 29.
The results were remarkable.
The WA2 case was identical to WA1 except that it had three additional mutations.</p>

<p><br />
<img src="/images/blog/ncov_nextstrain_2020_03_01_wa2.png" alt="" />
<br /></p>

<p>This tree structure is consistent with WA2 being a direct descendent of WA1.
If this virus arrived in Snohomish County in mid-January with the WA1 traveler from Wuhan and circulated locally for 5 weeks, we’d expect exactly this pattern, where the WA2 genome is a copy of the WA1 genome except it has some mutations that have arisen over the 5 weeks that separate them.</p>

<p>Again, this tree structure is consistent with a transmission chain leading from WA1 to WA2, but we wanted to assess the probability of this pattern arising by chance instead of direct transmission.
Scientists often try to approach this situation by thinking of a “null model”, ie if it was coincidence, how likely of a coincidence was it?
Here, WA1 and WA2 share the same genetic variant at site 18060 in the virus genome, but only 2/59 sequenced viruses from China possess this variant.
Given this low frequency, we’d expect probability of WA2 randomly having the same genetic variant at 2/59 = 3%.
To me, this not quite conclusive evidence, but still strong evidence that WA2 is a direct descendent of WA1.</p>

<p>Additional evidence for the relationship between these cases comes from location.
The Seattle Flu Study had screened viruses from all over the greater Seattle area, however, we got the positive hit in Snohomish County with cases less than 15 miles apart.
This by itself would only be suggestive, but combined with the genetic data, is firmer evidence for continued transmission.</p>

<p>I’ve been referring to this scenario as “cryptic transmission”.
This is a technical term meaning “undetected transmission”.
Our best guess of a scenario looks something like:</p>

<p><br />
<img src="/images/blog/ncov_transmission_chain_wa1_wa2.png" alt="" />
<br /></p>

<p>We believe this may have occurred by the WA1 case having exposed someone else to the virus in the period between Jan 15 and Jan 19 before they were isolated.
If this second case was mild or asymptomatic, contact tracing efforts by public health would have had difficulty detecting it.
After this point, community spread occurred and was undetected due to the <a href="https://web.archive.org/web/20200225155220/https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-criteria.html">CDC narrow case definition</a> that required  direct travel to China or direct contact with a known case to even be considered for testing.
This lack of testing was a critical error and allowed an outbreak in Snohomish County and surroundings to grow to a sizable problem before it was even detected.</p>

<p>Knowing that transmission was initiated on Jan 15 allows us to estimate the total number of infections that exist in this cluster today.
Our preliminary analysis puts this at <a href="https://twitter.com/trvrb/status/1234589598652784642">570 with an 90% uncertainty interval of between 80 and 1500 infections</a>.</p>

<p>Back on Feb 8, <a href="https://twitter.com/trvrb/status/1226241415522373632">I tweeted this thought experiment</a>:</p>

<p><img src="/images/blog/ncov_seeding.jpg" alt="" />
<br /></p>

<p>We know that Wuhan went from an index case in ~Nov-Dec 2019 to several thousand cases by mid-Jan 2020, thus going from initial seeding event to widespread local transmission in the span of ~9-10 weeks.
We now believe that the Seattle area seeding event was ~Jan 15 and we’re now ~7 weeks later.
I expect Seattle now to look like Wuhan around ~1 Jan, when they were reporting the first clusters of patients with unexplained viral pneumonia.
We are currently estimating ~600 infections in Seattle, this matches my <a href="https://bedford.io/projects/ncov-phylodynamics/">phylodynamic estimate of the number of infections in Wuhan on Jan 1</a>.
Three weeks later, Wuhan had thousands of infections and was put on <a href="https://www.nytimes.com/2020/01/22/world/asia/china-coronavirus-travel.html">large-scale lock-down</a>.
However, these large-scale non-pharmaceutical interventions to create social distancing had a huge impact on the resulting epidemic.
China averted many millions of infections through these intervention measures and <a href="https://covid2019.azurewebsites.net/">cases there have declined substantially</a>.</p>

<p><img src="/images/blog/ncov_hubei_cases_2020_03_02.png" alt="" />
<br /></p>

<p>This suggests that this is controllable.
We’re at a critical junction right now, but we can still mitigate this substantially.</p>

<p>Some ways to implement non-pharmaceutical interventions include:</p>

<ul>
  <li>Practicing social distancing, such as limiting attendance at events with large groups of people</li>
  <li>Working from home, if your job and employer allows it</li>
  <li>Staying home if you are feeling ill</li>
  <li>Take your temperature daily, if you develop a fever, self-isolate and call your doctor</li>
  <li>Implementing good hand washing practices - it is extremely important to wash hands regularly</li>
  <li>Covering coughs and sneezes in your elbow or tissue</li>
  <li>Avoiding touching your eyes, nose, and mouth with unwashed hands</li>
  <li>Disinfecting frequently touched surfaces, such as doorknobs</li>
  <li>Beginning some preparations in anticipation of social distancing or supply chain shortages, such as  ensuring you have sufficient supplies of prescription medicines and ensuring you have about a 2 week supply of food and other necessary household goods.</li>
  <li>With these preparation in mind, it is important to not panic buy. Panic buying unnecessarily increases strain on supply chains and can make it difficult to ensure that everyone is able to get supplies that they need.</li>
</ul>

<p>For more information please see:</p>

<ul>
  <li><a href="https://www.kingcounty.gov/depts/health/communicable-diseases/disease-control/novel-coronavirus.aspx">Public Health Seattle and King County resource center for COVID-19</a></li>
  <li><a href="https://www.doh.wa.gov/Emergencies/Coronavirus">Washington State Department of Health resource center for COVID-19</a></li>
  <li><a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">US Centers for Disease Control and Prevention – COVID-19 resource page</a></li>
</ul>

			</div>
		</div>
		<div class="col-lg-1"></div>
	</div>

	<div class="bigspacer"></div>
	<div class="spacer"></div>

</div>


	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
